src/templates/static/affiliates/signup.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Table } from '../../_common/components/elements.jsx';
 import { TabContainer, TabContent, TabContentContainer, TabsSubtabs } from '../../_common/components/tabs.jsx';
 
-const BoxRow = ({ children, top_row, bottom_row }) => (
-    <div className={`box-row gr-row ${top_row ? 'box-row-top' : ''} ${bottom_row ? 'box-row-bottom' : ''}`}>
+export const BoxRow = ({ children, top_row, bottom_row, no_border }) => (
```

Comment:
not sure if it's fine to keep these element definitions here or it's better to move them to a more common folder and import it in both affiliates and ib pages. up to you, I'm okay with both

Reply:
@negarn component uses hardcoded path for `images/pages/affiliates`, so I think it's better to keep it here. I don't think it can be easily reused on other pages.

Reply:
sounds good to me.